stdlib/socket/src/main/ballerina/socket/Module.md

```diff
@@ -0,0 +1,63 @@
+## Module overview
+This module provides an implementation for connecting to remote socket server or act as a server for incoming socket request. The module facilitates two types of endpoints as `Client` and `Listener`.
+## Samples
+### Listener endpoints
+The sample given below shows how a listener is used to listen to the incoming socket request. The `onAccept(socket:Caller)` resource function gets invoked when a new client connected. New client will be represented using the `socket:Caller`.
```

Comment:
Change to "... The `onAccept(socket:Caller)` resource function is invoked when a new client is connected. The new client is represented using the `socket:Caller`."